package.json

```diff
@@ -12,6 +12,10 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
```

Comment:
Instead of doing this as part of the build process, can the node module be directly packaged or submoduled to?
Right now, Cloning the git repo makes a working copy of brackets. This will mean that we have to do a npm install for brackets to work right?